frontend/src/apis/books/BooksAPI.tsx

```diff
@@ -49,7 +49,7 @@ export interface APIBook {
 
 export enum APILineItemType {
   PURCHASE_ORDER = "purchase order",
-  SALES_RECONCILIATION = "sales reconciliation",
+  SALES_RECORD = "sales reconciliation",
```

Comment:
The string here needs to be changed

Reply:
The string cannot be changes as Daniel wanted to maintain keeping sales reconciliation as the wording still used for the API requests. This includes the string for types. Backend still sends sales reconciliation.